ui/src/components/HelpAccordian.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react";
+import styled from "styled-components";
+import { colors } from "../styles/colors";
+import Accordion from "@material-ui/core/Accordion";
+import AccordionSummary from "@material-ui/core/AccordionSummary";
+import AccordionDetails from "@material-ui/core/AccordionDetails";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { Body1 } from "../styles/typography";
+
+const StyledAccordianSummary = styled(AccordionSummary)`
+  flex-direction: row-reverse;
+`;
+
+const StyledBox = styled.div`
+  background: ${colors.neutral};
+  border-radius: 10px;
+  padding: 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledAccordian = styled(Accordion)`
+  && {
+    box-shadow: none;
+  }
+
+  .MuiAccordionSummary-root {
+    padding: 0px;
+  }
+
```

Comment:
Not sure which class to change here but can you add an `overflow: hidden` to get rid of the horizontal scroll?

![image](https://user-images.githubusercontent.com/19617248/101396744-cae3a380-3899-11eb-9e2d-a70529696ce7.png)